Squirrel/Controllers/AccountController.cs

```diff
@@ -1,11 +1,17 @@
 using Microsoft.AspNetCore.Mvc;
+using Squirrel.Contexts;
 
 namespace Squirrel.Controllers
 {
     [ApiController]
     [Route("api/[controller]")]
     public class AccountController : Controller
     {
+        private readonly ApplicationContext context;
```

Comment:
Private/internal fields are supposed to be named with _
```suggestion
        private readonly ApplicationContext _context;
```